### ISSUE TITLE
Loading images from URL

### ISSUE THREAD

**Issue by blowsie** [destinationstransfers/passkit#19]
node-passbook mentions you can load images from URL.
I noticed you had removed this from your documentation so I assume that you don't directly support it.

What would be the best approach for loading the images from a URL?

Im currently loading the images using request and buffer, but it feels a bit heavy.

```
 pass.images.setImage('icon', '1x', function(err, callback) {
            request({
              url: ENDPOINTS.IMAGE + offer.agency.image + '?w=29&h=29&bgcolor=ffffff',
              encoding: null,
              method: 'GET',
              proxy: null
            }, function(error, response, body) {
              callback(body);
            })
          }
        );
```

**Comment by tinovyatkin** [comment]
Yes, I removed remote images downloading part, because it was implemented at very simple way that in production may result in a lot of hard to find bugs. 
Remember, that Pass has some required images and whole pass creation will fail if you will fail to retrieve such required image (`icon` from your example, is required).
So, if somebody will FORCE ME to use remote images in pass creation I will at least use an HTTP client with retry and local caching. So far, I know just one - https://github.com/zkat/make-fetch-happen

Then, I personally will do something like `async/await` and buffers:

```js
async function loadRemoteImages(pass, ...images) {
   ...
  const res = await fetch(url, ...);
  assert(res.ok);
  const imgBuffer = res.buffer();
  pass.images.setImage('icon', '1x', imgBuffer);
}
```
